vyked/registry.py

```diff
@@ -499,6 +504,11 @@ def _show_current_state(self, protocol):
                        'XSubscription List': self._repository._subscribe_list,
                        'Service Dependencies': self._repository._service_dependencies}
         protocol.send(status_dict)
+        
+    def _remove_service_from_xsubscribe(self, packet, protocol):
+        params = packet['params']
+        self._repository._remove_service_from_xsubscribe(params['service'], params['version'])
```

Comment:
.lower() should be used I guess for service

Reply:
we can used but maintaining consistency as lower is not used in many packets type, 
though we can change that